.golangci.yaml

```diff
@@ -0,0 +1,13 @@
+
+linters:
+  enable:
+    - unused
+    - prealloc
+    - gofmt
+    - asasalint
+    - copyloopvar
+    - nlreturn
```

Comment:
Why did you choose only this linters?

Reply:
To tell the truth, idk) It's combination of improvization & few common practices i heard of.

Can you advise some more to choose?

Reply:
In our project linters is not something static so it get bigger number of rules in over time.
Linters at first should close your pains working with code =)

Here is not bad template https://gist.github.com/maratori/47a4d00457a92aa426dbd48a18776322 with descriptive comments where you can look up smth new for you.